README.md

```diff
@@ -149,7 +149,7 @@ const html = Tram.html()
 // create a custom element to display a color option
 const colorElement = (attrs, children) => {
   return html`
-    <button onclick=${attrs.onclick}>${children}</button>
+    <button onclick=${attrs.onclick}>${children} />
```

Comment:
I think we need to keep this ending tag

Reply:
Whoops! Thanks for catching this one

Reply:
NP, when you get a chance, just add a commit to this branch to re-add the closing tag.